### ISSUE TITLE
Where is the appendix or supplementary material of this paper?

### ISSUE THREAD

**Issue by machanic** [Sandy-Zeng/NPAttack#1]
I download your paper from the ECCV official website. But it does not include the supplementary material.
Can you provide the complete version of your paper (including the supplementary material) on the arXiv?
Thank you!

**Comment by Sandy-Zeng** [comment]
Thank you for your attention to our work. Here is the arXiv link of our paper: https://arxiv.org/abs/2009.11508.

**Comment by Sandy-Zeng** [comment]
Thank you for your attention to our work. Here is the arXiv link of our paper: https://arxiv.org/abs/2009.11508.

**Comment by machanic** [comment]
@Sandy-Zeng Your paper is excellent, but hard to understand from the begining. Can you explain a bit more so that a reader can easier to understand? Or if you can point out the core code of the main logic so that I can quickly read the main point. 
Thank you very much.

**Comment by bymavis** [comment]
@machanic Yes. The main idea is that we would like to improve query efficiency by optimizing the distribution of adversarial examples/perturbations in the latent space. But the difficulty is that the normal auto encoder model, VAE for example, shares the same latent space on the whole dataset. That is, if optimized the mean/variance in the latent space in those VAE like models, they will suffer model collapse. The core code could be we optimize \mu, \delta in the latent variables $z$ and $r$, with the pre-trained NP model. Then we reconstruct adversarial examples using only the decoder part of such NP model.

**Comment by Sandy-Zeng** [comment]
@machanic Please refer to https://github.com/Sandy-Zeng/NPAttack/blob/master/NPAttack_untargeted.py#L91 for the core code of the main logic where the variable mu1 and s_context indicate the latent variables $z$ and $r$ in our paper.

**Comment by machanic** [comment]
Thank  you very much!  I will re-read your paper and your code and come back to ask you if neccessary.

**Comment by machanic** [comment]
@bymavis Can you write a blog to introduce this paper? Chinese is also OK language.
